dropwizard-auth/pom.xml

```diff
@@ -38,5 +38,10 @@
             <artifactId>jersey-test-framework-provider-grizzly2</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
You can remove this dependency, because Hamcrest is already imported to the module via JUnit.